Gems/LmbrCentral/Code/include/LmbrCentral/Shape/ShapeComponentBus.h

```diff
@@ -31,6 +31,23 @@ namespace LmbrCentral
         ShapeChange ///< The cache is invalid because the shape configuration/properties changed.
     };
 
+    /// Feature flag for work in progress on shape component translation offsets (see https://github.com/o3de/sig-simulation/issues/26).
+    constexpr AZStd::string_view ShapeComponentTranslationOffsetEnabled = "/Amazon/Preferences/EnableShapeComponentTranslationOffset";
+
+    /// Helper function for checking whether feature flag for in progress shape component translation offsets is enabled.
+    /// See https://github.com/o3de/sig-simulation/issues/26 for more details.
+    inline bool IsShapeComponentTranslationEnabled()
+    {
+        bool isShapeComponentTranslationEnabled = false;
+
+        if (auto* registry = AZ::SettingsRegistry::Get())
+        {
+            registry->Get(isShapeComponentTranslationEnabled, ShapeComponentTranslationOffsetEnabled);
+        }
```

Comment:
suggestion: I think you can just return here and not use an out param (see ViewportSettings.h/cpp for some examples)

Reply:
I think there's a function in ViewportSettings.h to support that, which I'd need to copy or move to the registry settings to make that work. I also don't really see the problem with the current version.

Reply:
Sure no worries just a random suggestion to streamline the code but this is fine as-is